dev-requirements.txt

```diff
@@ -10,5 +10,5 @@ cython
 sphinx
 sphinx-rtd-theme
 mypy==0.770
-click
+click>=8.0.0
```

Comment:
click 8.0.0 is need to allow slim-metadata parameter to be used as a flag with a default but also a flag that a user can set the value for.